checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/api/configuration/CardConfigurationFactories.kt

```diff
@@ -0,0 +1,39 @@
+package com.worldpay.access.checkout.api.configuration
+
+import com.worldpay.access.checkout.Card
+import com.worldpay.access.checkout.api.Callback
+import com.worldpay.access.checkout.logging.LoggingUtils
+import com.worldpay.access.checkout.model.CardConfiguration
+import com.worldpay.access.checkout.validation.AccessCheckoutCardValidator
+
+/**
+ * Creates a [CardConfigurationClient] instance
+ */
+internal object CardConfigurationClientFactory {
+
+    /**
+     * @return a [CardConfigurationClient] instance
+     */
+    @JvmStatic
+    internal fun createClient(): CardConfigurationClient {
+        return CardConfigurationClientImpl()
+    }
+}
+
```

Comment:
```suggestion

/**
 * Responsible for loading the remote card configuration file that is hosted by Access Worldpay
 * and used for driving the card validation logic. When the card configuration has been
 * returned, it will then be set on the `CardValidator` assigned to the `Card` so that the
 * fields will start validating.
 */
```